### DISCUSSION TITLE
Can I sychronize linux machines with lldap?

### DISCUSSION THREAD

**Discussion by theAlevan** [lldap/lldap#433] [Q&A] [answered]
I am a happy user of lldap and a begginer in ldap. I host multiple linux VM's on my server.

Can I connect them to lldap to synchronize the users and groups (so if I create a user in lldap, the user get propagated to the VMs)?
I don't want to spend hours of research into it if it's not possibile but on a full fledged ldap implementation.

**Comment by nitnelave** [comment, marked as answer]
I think you'll be interested in following https://github.com/nitnelave/lldap/issues/294

But short story: not yet, but it's planned.
